app/src/main/kotlin/com/wire/android/ui/theme/WireColorScheme.kt

```diff
@@ -118,6 +119,7 @@ private val LightWireColorScheme = WireColorScheme(
     ongoingCallBackground = WireColorPalette.Gray90,
     initiatingCallBackground = WireColorPalette.Gray20,
     callingIncomingBackground = WireColorPalette.Gray20,
+    groupAvatarColors = listOf(WireColorPalette.LightRed500, WireColorPalette.LightBlue500, WireColorPalette.LightGreen500, WireColorPalette.LightPurple500)
```

Comment:
According to design documentation, there are 6 "main" colors (Blue500, Green500, Petrol500, Purple500, Red500, Amber500), is it specified which ones can be used for group conversations or should we use all 6?